### PR TITLE
Initial Draft Febuary 24 2024 Meeting minutes

### PR DESCRIPTION
**Pull request by timlegge**

### CODE REVIEW

**Comment by timlegge** [comment]
@sjn do you want me to merge this?

**Review by sjn — APPROVED** [review]
LGTM

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+1. No agenda :-)
+2. If you're a newcomer, please come early so you can introduce yourself, and we ourselves!
+
+## 15:00 UTC – Meeting begin (~~timeboxed~~)
+
+### Welcome – @sjn
+
+* @sjn welcomed all in @oalders absence
+* @timlegge took minutes
+* Attendees: @stigo, @tux, @sjn, @ingy, @timlegge, Tina Müller
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:25]:
```suggestion
* Attendees: stigo, @tux, @sjn, @ingy, @timlegge, Tina Müller
```

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+3. [CPAN Provenance & Supply Chain Security](https://github.com/orgs/CPAN-Security/projects/3)
+4. [CPAN Security Outreach & Information](https://github.com/orgs/CPAN-Security/projects/12)
+    * @sjn SBOM draft branch is describing SBOM information as a WIP.
+    * @sjn looking at the language that other groups are using and the terms in use. Proposing some new naratives. Eg "3rd party components" should be called "2nd party" for opensource components
+    * @tux - advise to end users about the issues, you might be vulnerable - how SBOM helps, how to fix software, give users a path
+    * @tina muller maintains a script to extract SBOM data from a package at suse
+        * [openSUSE devel:languages:perl](https://build.opensuse.org/project/show/devel:languages:perl)
+        * [All CPAN packages starting with A](https://build.opensuse.org/project/show/devel:languages:perl:CPAN-A) 
+    * @sjn: get metadata from a CPAN package to get the dependencies for use in determining dependencies, etc. 
+    * @stigo: are distro packaging details in scope for SBOM security for us?
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:50]:
```suggestion
    * stigo: are distro packaging details in scope for SBOM security for us?
```

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        - @sjn WIP: simple majority for most things, charter changes require more agreement (add particulars) 
+
+### Ongoing Discussions
+
+1. SBOM generation (PAUSE, Dist::Zilla, both?)
+    - @sjn some things at the author level, some tooling is needed and will likely be used as well as pause and metacpan (final pkg url).
+    - @sjn volunteered to provide a session to demonstrate/discuss what is required in an SBOM file
+2. Moving to cpan-security-advisory YAML format v2 (brian d foy email)
+    - @tux had a discussion with brian regarding the use of his Test module.  
+    - @stigo question on how to consume the data -
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:67]:
```suggestion
    - stigo question on how to consume the data - 
```

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    - @tux is open to changing the module to an api versus the data direct from brian's repo.  
+    - @tux is looking ta the RedHat advisory and the formats of the data - it is a work in progress.  
+    - @tux would like to read the info and give back the information is a standard format in the future.  Net::CVE gives you everything you need to interact.
+    - @svn - end users want the specific line number and file that a vulnerability is found in to help find out whether they are vulnerable
+3. Group Name (last meeting's change)
+    - @garu discussed on IRC - surprised by the group name change- should we change (working group is a well known group).
+    - Should we change the group name back.
+    - Table it till the next meeting.
+4. Adding CVEs to CPANSA- vulns that are missing them
+    - @stigo gone through the CPANSA database registered 2 CVEs for old vulnerabilities - Debian follows the CVE database.  3 Mojolicious vulnerabilities and others that need CVEs (until 2020, should maybe also include 2018).
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:77]:
```suggestion
    - stigo gone through the CPANSA database registered 2 CVEs for old vulnerabilities - Debian follows the CVE database.  3 Mojolicious vulnerabilities and others that need CVEs (until 2020, should maybe also include 2018).
```

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    - Deciding if we should have our own vuln identifiers or use CVEs
+    - @stigo initial discussion was to have our own ID - thinking we simply use the CVE database for everything we know.
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:79]:
```suggestion
    - stigo - initial discussion was to have our own ID - thinking we simply use the CVE database for everything we know.  
```

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+    - Table it till the next meeting.
+4. Adding CVEs to CPANSA- vulns that are missing them
+    - @stigo gone through the CPANSA database registered 2 CVEs for old vulnerabilities - Debian follows the CVE database.  3 Mojolicious vulnerabilities and others that need CVEs (until 2020, should maybe also include 2018).
+    - Deciding if we should have our own vuln identifiers or use CVEs
+    - @stigo initial discussion was to have our own ID - thinking we simply use the CVE database for everything we know.  
+    - Possibly a temp ID?  The IDs int he CPANSA ids have been changing over time.  
+    - CVEs are the well known identifier.
+    - Go forward with CVE untill we need something else?
+5. Checking if we can get crypto functions into core
+    - @stigo what is the status - does anyone know?  Getting the verification functions would be ideal
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:84]:
```suggestion
    - stigo - does anyone know if this has been proposed?  Getting the verification functions would be ideal for TUF and similar
```

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+6. CVE advisories to MetaCPAN pages for distributions
+    - @stigo - nothing new but hope to get some things added to the pages
+7. Discuss the TUF implementation briefly
+    - @stigo - tuf implements repo and author signing - proper signing with attack mitigations - started looking at it but no progress - we need a CPAN implementation - for pause to do repo signing and possibly author. is CycloneDX related?
+    - @ingy - high level implementation is not large (maybe) - increases the profile and credibility of Perl and would be a good thing.
+    - @sjn - can @ingy and @stigo put together something in a project for what we need to do - will be very important for funding, etc.
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:90]:
```suggestion
    - @sjn - can @ingy and stigo put together something in a project for what we need to do - will be very important for funding, etc.
```

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+    - @stigo initial discussion was to have our own ID - thinking we simply use the CVE database for everything we know.  
+    - Possibly a temp ID?  The IDs int he CPANSA ids have been changing over time.  
+    - CVEs are the well known identifier.
+    - Go forward with CVE untill we need something else?
+5. Checking if we can get crypto functions into core
+    - @stigo what is the status - does anyone know?  Getting the verification functions would be ideal
+6. CVE advisories to MetaCPAN pages for distributions
+    - @stigo - nothing new but hope to get some things added to the pages
+7. Discuss the TUF implementation briefly
+    - @stigo - tuf implements repo and author signing - proper signing with attack mitigations - started looking at it but no progress - we need a CPAN implementation - for pause to do repo signing and possibly author. is CycloneDX related?
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:88]:
```suggestion
    - stigo - nothing new but hope to get some things added to the pages
7. Discuss the TUF implementation briefly
    - stigo - tuf implements repo and author signing - proper signing with attack mitigations - started looking at it but no progress - we need a CPAN implementation - for pause to do repo signing and possibly author. 
    - sjn: is CycloneDX related?
```

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+### Ongoing vulnerabilities
+
+1. Vince issue is being transfered to GitHub
+    - @stigo - not a vulnerability that really affects us (much)
+
+### Operating changes
+
+1. Proposal to use CC-BY-SA-4.0 as a content license in CPANSec for general web site content
+    - @timlegge, @stigo, @sjn vote to use CC-BY-SA-4.0
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:100]:
```suggestion
    - @timlegge, stigo, @sjn vote to use CC-BY-SA-4.0 
```

File: docs/minutes/cpansec-minutes-2024-02-24.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+1. Vince issue is being transfered to GitHub
+    - @stigo - not a vulnerability that really affects us (much)
+
+### Operating changes
+
+1. Proposal to use CC-BY-SA-4.0 as a content license in CPANSec for general web site content
+    - @timlegge, @stigo, @sjn vote to use CC-BY-SA-4.0 
+    - ingy abstain
+    - motion carried
+    - @stigo - clarified that we need to exclude vulnerability info as it needs to be public info
```
Comment (stigtsp) [inline docs/minutes/cpansec-minutes-2024-02-24.md:103]:
```suggestion
    - stigo - we should scheck that CC-BY-SA-4.0 is compatible for vulnerability info
```

**Review by stigtsp — COMMENTED** [review]
Corrected some instances where the @-tagging points to a different user, but not doing it

Clarified some points

**Pull request merged**